Purchases/Logging/Strings/PurchaseStrings.swift

```diff
@@ -163,6 +167,15 @@ extension PurchaseStrings: CustomStringConvertible {
                 "Verification error \(errorMessage)"
         case .begin_refund_request_unsupported:
             return "Tried to call beginRefundRequest in a platform that doesn't support it!"
+        case .begin_refund_for_entitlement_nil_customer_info(let entitlementID):
+            return "Failed to get entitlement\( entitlementID ?? "") for refund. " +
```

Comment:
need a space between `entitlement` and `\( entitlementID ?? "")`

Reply:
i was hoping to get the space within the () because if entitlementID is null, we don't want the space.. but i'm not sure i ended up testing that out...

Reply:
That makes sense, but since it's part of an expression and not part of the string, it won't work 😄
Here's how I would do it:
![Screen Shot 2021-12-22 at 9 28 35 AM](https://user-images.githubusercontent.com/1304821/147132053-bc5b6cf9-4703-43e8-ae1a-351958250f83.png)